lib/src/query_component.dart

```diff
@@ -45,17 +49,18 @@ class QueryComponent {
   String insertionOrderId;
   bool highlightUnderpacing = false;
 
-  QueryComponent(this._queryService, this._excel);
-
-  bool get isAdvertiserQuery => _queryType == QueryType.byAdvertiser;
-  set isAdvertiserQuery(bool checked) =>
-      checked ? _queryType = QueryType.byAdvertiser : null;
+  // Radio button states with byAdvertiser selected as default.
+  RadioButtonState advertiserQuery = RadioButtonState(true, 'byAdvertiser');
```

Comment:
Seems like the radio button can only take string values :(

Saw [other people](https://github.com/dart-lang/angular/issues/293) complain about this too.